cloudpickle/cloudpickle.py

```diff
@@ -577,6 +591,7 @@ def save_function_tuple(self, func):
         if hasattr(func, '__qualname__'):
             state['qualname'] = func.__qualname__
         save(state)
+        save(self.override_existing_globals)
```

Comment:
Could you put this variable inside `state`? Because in the future there will could be more switches and when the argument count reaches 5 there will be no way to tell it from the older implementation.